cdap-ui/app/directives/widget-container/widget-ds-multiplevalues/widget-ds-multiplevalues.js

```diff
@@ -63,8 +63,9 @@ angular.module(PKG.name + '.commons')
           var str = '';
 
           angular.forEach($scope.properties, function(p) {
-            var isAnyEmptyValue = p.values.filter(function(val) { return val.length === 0; });
-            if (isAnyEmptyValue.length) { return; }
+            // If any of the configuration field is filled then do not remove it.
+            var isAnyEmptyValue = p.values.filter(function(val) { return val.length !== 0; });
+            if (isAnyEmptyValue.length === 0) { return; }
```

Comment:
@neelesh-nirmal if you want to allow the blank value then below lines are not required 

var isAnyEmptyValue = p.values.filter(function(val) { return val.length !== 0; });
            if (isAnyEmptyValue.length === 0) { return; }